lib/Makefile

```diff
@@ -71,9 +71,9 @@ obj-$(CONFIG_ECDSA) += ecdsa/
 obj-$(CONFIG_$(SPL_)RSA) += rsa/
 obj-$(CONFIG_HASH) += hash-checksum.o
 obj-$(CONFIG_BLAKE2) += blake2/blake2b.o
-obj-$(CONFIG_SHA1) += sha1.o
-obj-$(CONFIG_SHA256) += sha256.o
-obj-$(CONFIG_SHA512) += sha512.o
+obj-$(CONFIG_$(SPL_)SHA1) += sha1.o
```

Comment:
This doesn't enable SHA1 in SPL, it just allows enabling it in SPL separately :)

Reply:
@Tim-Anderson we set the maximum size of SPL in `lmp.cfg` including a space for a sign so that SPL can't be built at all if there is no space for a sign.